x-pack/solutions/search/test/serverless/functional/test_suites/navigation.ts

```diff
@@ -249,7 +250,7 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
           'management:trained_models',
           'management',
           'cloudLinkDeployment',
-          'cloudLinkBilling',
+          // 'cloudLinkBilling', // Billing is not shown for developer role
```

Comment:
Is it shown for the admin role? Just checking to make sure it can be easily reproduced locally.

Reply:
maybe search should adjust the test to use admin role